cmd/flags_header.go

```diff
@@ -12,8 +12,8 @@ import (
 )
 
 var (
-	headersTrustedHashFlag = "headers.trusted-hash"
-	headersTrustedPeerFlag = "headers.trusted-peer"
+	headersTrustedHashFlag  = "headers.trusted-hash"
+	headersTrustedPeersFlag = "headers.trusted-peers"
```

Comment:
This is a breaking change so we should add the proper entry to changelog

Reply:
done